docker/py3-primary/Dockerfile

```diff
@@ -0,0 +1,120 @@
+FROM demisto/mlurlphishing:1.0.0.28347 as mlurlphishing
+
+FROM devdemisto/py3-tools-ubi:1.0.0.37605
+
+COPY requirements.txt .
+
+# chromium
+COPY chromium/download_chromedriver.sh .
+COPY repos/google-chrome.repo /etc/yum.repos.d/
+
+# will always install the latest stable google-chrome version
+RUN dnf update --nodocs -y  \
+    && dnf upgrade -y --nodocs \
+    && dnf install -y wget \
+    && wget https://dl.google.com/linux/linux_signing_key.pub \
+    && wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official \
+    && rpm --import linux_signing_key.pub \
+    && rpm --import RPM-GPG-KEY-CentOS-Official \
+    && dnf install -y --nodocs python3-devel gcc gcc-c++ make wget git unzip llvm-libs google-chrome-stable \
```

Comment:
google-chrome-stable will take from google-chrome in the repo folder?

Reply:
exactly